website/docs/artist_hosts_nuke_tut.md

```diff
@@ -0,0 +1,166 @@
+---
+id: artist_hosts_nuke_tut
+title: Nuke QuickStart
+sidebar_label: Nuke QuickStart
+---
+
+# Nuke QuickStart
+- [Nuke QuickStart](#nuke-quickstart)
+  - [Run Nuke - Shot and Task Context](#run-nuke---shot-and-task-context)
+  - [Save Nuke script – the Work File](#save-nuke-script--the-work-file)
+  - [Load plate – Asset Loader](#load-plate--asset-loader)
+  - [Create Write Node – Instance Creator](#create-write-node--instance-creator)
+  - [What Nuke Publish Does](#what-nuke-publish-does)
+      - [Publish steps](#publish-steps)
+      - [Pyblish Note and Intent](#pyblish-note-and-intent)
+      - [Pyblish Checkbox](#pyblish-checkbox)
+      - [Pyblish Dialog](#pyblish-dialog)
+  - [Review](#review)
+  - [Render and Publish](#render-and-publish)
+  - [Version-less Render](#version-less-render)
+  - [Fixing Validate Containers](#fixing-validate-containers)
+  - [Fixing Validate Version](#fixing-validate-version)
+
+This QuickStart is just a small introduction to what OpenPype can do for you. It attempts to make an overview for compositing artists, and simplifies processes that are better described in specific parts of the documentation.
+
+## Run Nuke - Shot and Task Context
+OpenPype has to know what shot and task you are working on. You need to run Nuke in context of the task, using Ftrack Action or OpenPype Launcher to select the task and run Nuke.
+
+![Run Nuke From Ftrack](assets/nuke_tut/nuke_RunNukeFtrackAction_p2.png)
+![Run Nuke From Launcher](assets/nuke_tut/nuke_RunNukeLauncher_p2.png)
+
+
+OpenPype menu shows the current context - shot and task.
+
+If you use Ftrack, executing Nuke with context stops your timer, and starts the Ftrack clock on the shot and task you picked.
+
+![Context](assets/nuke_tut/nuke_Context.png)
+
+:::tip Admin Tip - Nuke version
+You can [configure](admin_settings_project_anatomy.md#Attributes) which version(s) will be available for current project in **Studio Settings → Project → Anatomy → Attributes → Applications**
+:::
+
+## Save Nuke script – the Work File
+Use OpenPype - Work files menu to create a new Nuke script. Openpype offers you the preconfigured naming. 
+![Context](assets/nuke_tut/nuke_AssetLoader.png)
+
+Openpype makes initial setup for your Nuke script. It is the same as running [Apply All Settings](artist_hosts_nuke.md#apply-all-settings) from the OpenPype menu.
+Reads frame range and resolution from Avalon database, sets it in Nuke Project Settings
+Creates Viewer node, sets it’s range and indicates handles by In and Out points
+Reads Color settings from the project configuration, and sets it in Nuke Project Settings and Viewer
+Sets project directory in the Nuke Project Settings to the Nuke Script Directory
+
+:::tip Admin Tip - Workfile Naming
```

Comment:
Rather then multiple Tip frames it would be better to have one with multiple points to cover.

Reply:
OK, will do. Imho looks less appealing as one.